script.js

```diff
@@ -84,3 +84,28 @@ addBookForm.addEventListener('submit', (event) => {
 });
 
 bookCollection.loadFromLocalStorage();
+
+// Add event listeners to the links
+const allBooksLink = document.getElementById('all-books-link');
+const addBookLink = document.getElementById('add-book-link');
+const contactLink = document.getElementById('contact');
+const allBooksContainer = document.getElementById('all-books-container');
+const addBookContainer = document.getElementById('add-book-container');
+const about = document.getElementById('about');
+
+allBooksLink.addEventListener('click', () => {
+  allBooksContainer.style.display = 'block'; // Show all books container
+  addBookContainer.style.display = 'none'; // Hide add book container
+  about.style.display = 'none';
+});
+
+addBookLink.addEventListener('click', () => {
+  allBooksContainer.style.display = 'none'; // Hide all books container
+  addBookContainer.style.display = 'block'; // Show add book container
+  about.style.display = 'none';
+});
```

Comment:
- [ ] Great job adding the navigation links, but to make sure the app matches the requirements, only one page should be shown per each link. The first (home) page should only show the list of books (if any), if no book is added, the page should be blank.

![pages](https://github.com/Zel-hub7/Awesome_Books/assets/53124548/66859d1c-546b-45f6-ba8b-05d90ca5d527)